interface/new/new.php

```diff
@@ -91,14 +91,14 @@ function validate() {
     $('.datepicker').datetimepicker({
         <?php $datetimepicker_timepicker = false; ?>
         <?php $datetimepicker_showseconds = false; ?>
-        <?php $datetimepicker_formatInput = false; ?>
+        <?php $datetimepicker_formatInput = true; ?>
         <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
         <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
     });
     $('.datetimepicker').datetimepicker({
         <?php $datetimepicker_timepicker = true; ?>
         <?php $datetimepicker_showseconds = false; ?>
-        <?php $datetimepicker_formatInput = false; ?>
+        <?php $datetimepicker_formatInput = true; ?>
```

Comment:
Does this work? (ie. does the actual date with the time get saved correctly in the database?)

Reply:
I fixed the datetimepicker to work with format date https://github.com/openemr/openemr/pull/1393/files#diff-855a610b8e24aa94e2fcbd331ceec699
It supposed to work correctly.